core/chains/evm/orm_test.go

```diff
@@ -3,10 +3,12 @@ package evm_test
 import (
 	"testing"
 
-	"github.com/smartcontractkit/sqlx"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/guregu/null.v4"
 
+	"github.com/smartcontractkit/sqlx"
+
```

Comment:
**nit** remove this space

Reply:
this is a external library, even though it does start with `smartcontractkit`... hence why the white space here

Reply:
Thanks. I missed that detail